CMakeLists.txt

```diff
@@ -125,8 +125,8 @@ include_directories(
 
 ## Declare a C++ library
 add_library(lidarPointLib
-  src/lidarPoint.cpp
```

Comment:
What is the Lib folder for?

Reply:
lidarPoint is not a ros node, it's used like a library. I think there's a proper way to make ROS libraries and include them but I spent some time on it and couldn't find anything that helpful, so for the essence of time I left it for now and made an asana task. Basically, the idea is to separate library classes from the ros nodes.

Reply:
gotcha